services/docker-compose.yml

```diff
@@ -701,10 +701,22 @@ services:
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver.entrypoints=http
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver.priority=2
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver.middlewares=${SWARM_STACK_NAME}_gzip@docker, ${SWARM_STACK_NAME_NO_HYPHEN}_sslheader@docker, ${SWARM_STACK_NAME}_webserver_retry
-    networks:
+    networks: &webserver_networks
       - default
       - interactive_services_subnet
 
+  web-api:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Task.Slot}}"
+    environment:
+      <<: *webserver_environment
+      WEBSERVER_HOST: ${WEBAPI_HOST}
+      WEBSERVER_PORT: ${WEBAPI_PORT}
+
+    networks: *webserver_networks
```

Comment:
minor: I was wondering why this guy needs to talk with interactive services at all. do you need any connection to them?

Reply:
Let me answer with a question, why does the webserver needs access to this network? Or any of the other. AFAIK none of the interactive services should have access to the stack. That privilege is delegated to the dynamic-sidecar that acts as intermediary, right?
@GitHK can you please comment on this?

Reply:
@pcrespov so first of all it has access to that network because it needs to call the /retrieve entrypoint of the services (all of them, also the legacy ones). So until we stop using that endpoint? (maybe we do, and then you can safely remove it, if we don't you cannot remove it.)
Nevertheless it does not solve the issue that the wb-api-server might not need it.

Reply:
created an issue